src/main/java/com/example/tournamentMaker/team/TeamService.java

```diff
@@ -0,0 +1,63 @@
+package com.example.tournamentMaker.team;
+
+import com.example.tournamentMaker.statistics.FootballStatistics;
+import com.example.tournamentMaker.team.player.FootballPlayer;
+import com.example.tournamentMaker.team.player.PlayerRepository;
+import com.example.tournamentMaker.tournament.Tournament;
+import com.example.tournamentMaker.tournament.TournamentRepository;
+import com.example.tournamentMaker.tournament.enums.Sport;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+class TeamService {
+    private final TeamRepository teamRepository;
+    private final TournamentRepository tournamentRepository;
+    private final PlayerRepository playerRepository;
+
+    void createTeam(TeamRequest teamRequest) {
+        Optional<Tournament> tournament = tournamentRepository.findByName(teamRequest.getTournamentName());
+        tournament.ifPresentOrElse(t -> {
+            Sport sport = t.getSport();
+            switch (sport) {
+                case FOOTBALL -> {
+                    Team team = new Team(teamRequest.getTeamName(), t);
+                    FootballStatistics footballStatistics = new FootballStatistics(team, 0, 0, 0, 0, 0);
```

Comment:
zrobilbym metode ktora robi taki inicjalny obiekt z pustymi statystykami. Lepiej by to obrazowalo sytuacje niz 0 wstawione, typu magic number w kodzie